interface/frontend/src/components/open-image/ImageSeries.vue

```diff
@@ -109,7 +109,7 @@
     },
     mounted: function () {
       EventBus.$on('dicom-selection', (path) => {
-        this.preview.path = path
+        this.preview.paths = path
```

Comment:
Probably, should be `this.preview.paths = [path]` and later we'll change the `open-image\TreeView.vue` component to provide all paths when user clicks on folder. Currently user clicks on exact file and its path sent using this event.